packages/graphiql-toolkit/src/create-fetcher/lib.ts

```diff
@@ -76,13 +80,24 @@ export const createWebsocketsFetcherFromUrl = (
 ) => {
   let wsClient;
   try {
+    const { createClient } = require('graphql-ws') as {
+      createClient: typeof createClientType;
+    };
+
     // TODO: defaults?
     wsClient = createClient({
       url,
       connectionParams,
     });
     return createWebsocketsFetcherFromClient(wsClient);
   } catch (err) {
+    if (errorHasCode(err)) {
+      if (err.code === 'MODULE_NOT_FOUND') {
+        throw Error(
+          'You need to install the `graphql-ws` package to use websockets',
```

Comment:
We could be more specific here to educate where this error is coming from. (We don't expose the `createWebsocketsFetcherFromUrl` function publicly, you end up in this code path when passing a `subscriptionUrl` to `createGraphiQLFetcher`.)
```suggestion
          'You need to install the `graphql-ws` package to use websockets when passing a `subscriptionUrl`',
```